src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

Comment:
use transition only for transform and color here

Reply:
add transition-property as new line, but can`t add at one line due to it not work when more than one property (color, transform etc.) in line "transition"